CHANGELOG.md

```diff
@@ -7,6 +7,7 @@
 * [CHANGE] `ingestion_rate_limit` limit override has been removed in favour of `ingestion_rate_limit_bytes`.
   `ingestion_burst_size` limit override has been removed in favour of `ingestion_burst_size_bytes`.
   This is a **breaking change** to the overrides config section. [#630](https://github.com/grafana/tempo/pull/630)
+* [CHANGE] Rename auth.enabled to multitenancy.enabled and set it as false by default. This is not a breaking change, but auth.enabled has been set as deprecated. 
```

Comment:
Technically this is a breaking change if you were relying on auth/multitenancy being enabled by default. Let's list it here.

Also needs the `[#646](https://github.com/grafana/tempo/pull/646)` link

Reply:
Added both things to the changelog! :)